pkgs/development/compilers/ligo/default.nix

```diff
@@ -93,5 +96,6 @@ coq.ocamlPackages.buildDunePackage rec {
     license = licenses.mit;
     platforms = [ "x86_64-linux" ];
     maintainers = with maintainers; [ ulrikstrid ];
+    broken = true;
```

Comment:
?

Reply:
Apologies on this, I marked it as broken when testing other packages, I'll fix this up asap.